modules/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,151 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Image-based upgrade for {sno} cluster with the {lcao}
+
+From {product-title} 4.14.7, the {lcao} 4.15 provides you with an alternative way to upgrade the platform version of a {sno} cluster.
+The image-based upgrade is faster than the standard upgrade method and allows you to directly upgrade from {product-title} <4.y> to <4.y+2>, and <4.y.z> to <4.y.z+n>.
```

Comment:
```suggestion
The image-based upgrade is faster than the standard upgrade method and allows you to directly upgrade from an {product-title} minor version to a minor version two releases newer, for example from 4.13 to 4.15.
You can also use image-based upgrades to upgrade between z-stream releases, for example, from 4.15.1 to 4.15.10. 
```

Reply:
if specifying examples, I'd suggest using versions where LCA is supported / available in official catalogs to avoid any possible confusion

`for example from 4.13 to 4.15` maybe better to `for example from 4.14 to 4.16`

Reply:
Yes wasn't sure about supported versions thanks